### PR TITLE
Initial landing page files

### PR DESCRIPTION
**Pull request by flattumc**

### CODE REVIEW

**Comment by flattumc** [comment]
Added initial web files for the landing page. This includes the logo, mount Fugi img, and another image for later testing.

Files added:
* index.html
* styles.css

This is the first iteration of the landing page. Everything is housed a central "app" folder which will be used long term. 

Screenshot 2024-07-14 170305

**Comment by flattumc** [comment]
I've also included the first mock up of the planning page:

Screenshot 2024-07-14 170330

File: app/web/plan-a-trip.html
Comment (nswai983) [inline app/web/plan-a-trip.html:1]: Love these pages! The desires are great, and I think they really fit the theme of what we're going for.

I did notice on my laptop screen (2560x1600), the top text of the trip planning page overlaps with the headers. Would you be able to add some padding here to ensure they don't overlap? See image below for what I'm seeing.

tripPlannerOverlap

Comment (nswai983) [inline app/web/plan-a-trip.html:1]: Having a back button on this page would be good too.

**Review by nswai983 — CHANGES_REQUESTED** [review]
Love the pages overall! Have a few comments on the plan-a-trip.html, but otherwise looks good.

File: app/web/plan-a-trip.html
Comment (flattumc) [reply app/web/plan-a-trip.html:1]: App currently supports backwards navigation through header

Comment (flattumc) [reply app/web/plan-a-trip.html:1]: Updated media settings to account for different variations of zoom (90%,100%,110%,and 125%). This should help assist with padding issues seen above as well as future issues of overlap

**Review by nswai983 — APPROVED** [review]
Changes look great. The addition of accessibility settings makes sure nothing overlaps!

**Pull request merged**